### PR TITLE
dom-events - Frank Bishop

### PR DESCRIPTION
**Pull request by FrankBishop**
Screenshot (126)

### CODE REVIEW

File: dom-events/buttons.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+function handleClick(event) {
+  console.log('button clicked');
+  console.log(event);
+  console.log(EventTarget);
```
Comment (codingcodymiller) [inline dom-events/buttons.js:4]: This should be `event.target`

File: dom-events/buttons.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+}
+var button = document.querySelector('.click-button');
+button.addEventListener('click', handleClick);
+
+function handleMouseOver(event) {
+  console.log('button hovered');
+  console.log(event);
+  console.log(EventTarget);
```
Comment (codingcodymiller) [inline dom-events/buttons.js:12]: This should be `event.target`

File: dom-events/buttons.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+var hover = document.querySelector('.hover-button');
+hover.addEventListener('mouseover', handleMouseOver);
+
+function handleDoubleClick(event) {
+  console.log('button double clicked');
+  console.log(event);
+  console.log(EventTarget);
```
Comment (codingcodymiller) [inline dom-events/buttons.js:21]: This should be `event.target`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Frank! A couple notes here:

**Comment by FrankBishop** [comment]
Thanks for the feedback.  I fixed it and now the console output looks much better.
Screenshot (127)

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**